website/frontend/templates/docs/scripting.html

```diff
@@ -152,12 +218,12 @@ <h3 id="in_x_y">$in(x,y)</h3>
           </li>
         </ul>
 
-        <h3 id="inmulti_x_y">$inmulti(x,y)</h3>
+        <h3 id="inmulti_x_y">$inmulti(%x%,y)</h3>
 
         <ul>
           <li>Returns true if multi-value variable
-            <code>x</code> contains
-            <code>y</code>.
+            <code>x</code> contains exactly
```

Comment:
Darn - I was hoping I would get away with that and not have to put it in a separate ticket and separate PR. I'll move it if you insist.